tests-examples/demo-todo-app.spec.ts

```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
```

Comment:
were these tests intended to be here?

Reply:
oops, these were auto-generated. deleted!

Reply:
It looks like these were auto-generated again. I wonder if you need to disable something in order to prevent this from being created again